src/screens/connectors/LauncherView.js

```diff
@@ -73,20 +42,18 @@ class LauncherView extends Component {
 
   async initConnector() {
     const { client, launcherContext } = this.props
-    let result = null
-    let connector = embeddedConnectors[launcherContext.job.message.konnector]
```

Comment:
Didn't we conclude that we should keep the "local" connectors mecanism until we are able to easily declare them through a local cozy-stack (compatible with `build --watch`)?

Reply:
Honestly there is already an easy method to test them, it works well https://zoe-notes.mycozy.cloud/public/?sharecode=xmKI8NByNkbP

For real time usage, I can implement something but I need to see with connector team how they want to handle that.

Reply:
The real time scenario seems to be critical in my point of view. The connector team works with this feature on a daily basis.

Let @KillianCourvoisier, @LucsT and @doubleface confirm that we can merge this as they will be the first ones impacted.